src/app/campaign/store/models/flight.models.ts

```diff
@@ -50,6 +51,8 @@ export const docToFlight = (doc: HalDoc): Flight => {
   const flight = filterUnderscores(doc) as Flight;
   flight.startAt = utc(flight.startAt);
   flight.endAt = utc(flight.endAt);
+  // the cutoff is midnight the following day, but want it to appear as if the flight ends on the date it is set to stop serving
+  flight.endAtFudged = utc(flight.endAt).subtract(1, 'days');
```

Comment:
Okay, so if i'm following this ...

1. This is a one-time transform of endAt -> endAtFudged when the remote HalDoc comes in.
2. Form field is bound to endAtFudged.
3. On form change, the CampaignActionService transforms endAtFudged back into endAt, which gets sent on any remote requests.  And we don't get an infinite loop since (1) only happens after a remote load.

Reply:
Yes! I had kind of thought at first we would hide it in the service layer, but I did that and it felt gross. And the dashboard isn't using the same service layer anyway.
(1) will happen on load and on flight form update, but yeah those updates should not be emitting any further updates.

I am still seeing a separate bug though where we do emit a flight form update on flight tab navigation, which causes a flight preview load.